rocAL/rocAL/source/rali_api_augmentation.cpp

```diff
@@ -60,11 +60,47 @@ THE SOFTWARE.
 #include "meta_node_rotate.h"
 #include "meta_node_ssd_random_crop.h"
 #include "meta_node_flip.h"
+#include "node_sequence_rearrange.h"
 
 #include "commons.h"
 #include "context.h"
 #include "rali_api.h"
 
+RaliImage  RALI_API_CALL
+raliSequenceRearrange(
+            RaliContext p_context,
+            RaliImage p_input,
+            unsigned int* new_order,
+            unsigned int  new_sequence_length,
+            unsigned int sequence_length,
+            bool is_output )
+{
+    if(!p_input || !p_context)
+        THROW("Null values passed as input")
+    Image* output = nullptr;
+    auto context = static_cast<Context*>(p_context);
+    try
+    {
+        auto input = static_cast<Image*>(p_input);
+        unsigned sequence_count = context->internal_batch_size() / sequence_length;
+        context->master_graph->set_sequence_rearrange_batch_decrementer(context->master_graph->user_batch_size());
```

Comment:
We had similar codacy warning before which was fixed by checking for p_context in the beginning. Please check with @shobana-mcw and fix this warnings

Reply:
In line 78 we are checking if both p_context and p_input is NULL. Do we need to check after dereferencing too? 
In our previous PR too we faced this warning which we decided to ignore since we are checking once we enter the function.

Reply:
Ok. Ignore this